src/components/KeyValProp/KeyValProp.tsx

```diff
@@ -0,0 +1,29 @@
+import { Typography, TypographyProps } from '@material-ui/core';
+import { FC } from 'react';
+import * as React from 'react';
+
+export type KeyValPropProps = TypographyProps & {
+  label?: string;
+  LabelProps?: TypographyProps;
+  value?: string | number | null;
```

Comment:
We could probably just make this a `ReactNode`.

Reply:
Changed to ReactNode